recipes-kernel/linux/linux-toradex-kmeta.inc

```diff
@@ -1,10 +1,13 @@
-SRCREV_meta = "99e38525088a105d6e7d99bd0eb07c9880606d64"
-SRCREV_meta:use-head-next = "${AUTOREV}"
+SRCREV_torizon-meta = "99e38525088a105d6e7d99bd0eb07c9880606d64"
+SRCREV_torizon-meta:use-head-next = "${AUTOREV}"
 
-KMETABRANCH = "kirkstone-6.x.y"
-KMETA = "kernel-meta"
+KMETABRANCH = "scarthgap-7.x.y"
```

Comment:
I merged the internal PR we had. But I don't see the scarthgap branch on our external repo here: https://github.com/toradex/toradex-kernel-cache

Maybe we need to do something to mirror this new branch?

Reply:
@leograba maybe we have to trigger some manual sync on GitHub?

Reply:
I'm a bit overloaded today so it may take a while, but it's at the top of my ToDo list. Will let you know as soon as possible.

Reply:
The branch is there now: https://github.com/toradex/toradex-kernel-cache/tree/scarthgap-7.x.y

What was required was to set the new branch as protected on GitLab. Then I manually triggered the mirroring so we didn't need to wait for the next automatic trigger.

What I *didn't* do is make this new branch the default branch either on GitLab or on the GitHub mirror. Please let me know if you'd like me to do it.

Reply:
Thanks @leograba!

>What I didn't do is make this new branch the default branch either on GitLab or on the GitHub mirror. Please let me know if you'd like me to do it.

This is ok. I think we could add this to whenever we do our first release, since at that point we'll switch our default branches.